### ISSUE TITLE
not responsive

### ISSUE THREAD

**Issue by julian-ca** [chenjiahan/vodal#13]
why is it not responsive?

**Comment by connecteev** [comment]
I agree, I just realized that vodal (as good as it is) is not responsive! We're in 2019 so this is a requirement for all web developers.

@chenjiahan Can we please add the ability for width and height to be specified as percentages? And also make vodal responsive by default?

```
                    <vodal measure="em" :show="show" animation="slideDown" :width="60" :height="27" :customStyles="{'box\-shadow': '0px 0px 30px 3px rgba(0, 0, 0, .8)'}" :closeOnEsc="true" :duration="600" className="my-dialog" @hide="hideModal">

```

**Comment by chenjiahan** [comment]
you can use measure props to specify that. The default behavior should not be changed because it will cause breaking change.

**Comment by connecteev** [comment]
@chenjiahan the measure props are in px only, that doesn't work for resizing the window in a "fluid" manner. We'll have to add multiple media queries...is it possible to add an option for '%' or 'em' to the measure props, so it is not a breaking change?

```
measure | string | px | measure of width and height
-- | -- | -- | --
```
